ui/components-react/src/test/filter-builder/FilterBuilderRuleGroup.test.tsx

```diff
@@ -130,122 +102,121 @@ describe("PropertyFilterBuilderRuleGroupRenderer", () => {
     getByDisplayValue("Prop");
   });
 
-  it("renders child rule group", () => {
-    const property: PropertyDescription = {
-      displayLabel: "Prop",
-      name: "prop",
-      typename: "int",
-    };
-    const { getByText } = renderWithContext(
-      <PropertyFilterBuilderRuleGroupRenderer
-        {...defaultProps}
-        group={{
-          id: "id",
-          operator: PropertyFilterRuleGroupOperator.And,
-          items: [
-            {
-              id: "childId",
-              groupId: "id",
-              operator: PropertyFilterRuleGroupOperator.Or,
-              items: [
-                {
-                  id: "grandChildId1",
-                  groupId: "childId",
-                  property,
-                },
-                {
-                  id: "grandChildId2",
-                  groupId: "childId",
-                  property,
-                },
-              ],
-            },
-          ],
-        }}
-      />
+  it("dispatches operator change event when operator is selected", async () => {
+    const actions = new PropertyFilterBuilderActions(sinon.spy());
+    const { container, findByText } = renderWithContext(
+      <PropertyFilterBuilderRuleGroupRenderer {...defaultProps} size="large" />,
+      { actions }
     );
+    const setRuleGroupOperatorSpy = sinon.stub(actions, "setRuleGroupOperator");
 
-    getByText(
-      TestUtils.i18n.getLocalizedString("Components:filterBuilder.operators.or")
+    const selector = container.querySelector<HTMLAnchorElement>(
+      ".fb-group-operator .iui-anchor"
```

Comment:
Do not use `ITwinUI` class names to select element (at least in new tests, older tests that uses `iui-*` class names should be fixed with https://github.com/iTwin/appui/pull/666). Select by text or use custom class name.